wordpress-mobile/WordPress-iOS
# Stats Insights Latest Post Summary: add alternate views

Fixes #10372 

This modifies what is displayed in the Latest Post Summary cell based on the post data.
- When the post has views, likes, or comments:
  - The data is shown.
  - The bottom has a 'View more' option.
- When the post has no views, likes, or comments:
  - The data is not shown.
  - The bottom has a 'Share Post' option.
- When the site has no posts:
  - The data is not shown.
  - The bottom has a 'Create Post' option.

For now, temporary alerts are shown when the following areas are tapped:
- Summary.
- View more.
- Share Post.
- Create Post.

To test:

---
Post has views, likes, or comments:
- Select a site that has a post with views, likes, or comments.
- Go to Stats > Insights.
- Verify the view is:
![view_more](https://user-images.githubusercontent.com/1816888/47940723-3a178e00-deb1-11e8-97d8-10758a8f2e9f.png)

---
Post has no views, likes, or comments:
- Select a site that has a post but no views, likes, or comments.
- Go to Stats > Insights.
- Verify the view is:
![share_post](https://user-images.githubusercontent.com/1816888/47940303-dc367680-deaf-11e8-8d0a-c3028deffd8c.png)

---
Site has no posts:
- Select a site with no posts.
- Go to Stats > Insights.
- Verify the view is:
![create_post](https://user-images.githubusercontent.com/1816888/47940407-19026d80-deb0-11e8-859e-5aae11eb2ec0.png)
